wiki/Contributing-to-the-Wiki.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to the Wiki
+If you're reading this, then you're interested in helping us document the system via the Wiki, thank you!
+We want to make sure that contributing to the Wiki is as easy as possible so that anyone that would like to help us with this task can do so. In that light, some guidelines to contribute to the Wiki are outlined below.
+
+## How the Wiki is updated
+Typically, a repository's Wiki can only be updated by the owners of that repo, this made community provided documentation cumbersome to include. In order to easily accept community provided documentation this repository is utilizing a [GitHub Action](https://github.com/Andrew-Chen-Wang/github-wiki-action) which will generate the Wiki from any Markdown files merged to the [`publish-wiki` branch's `/wiki/` folder](https://github.com/foundryvtt/dnd5e/tree/publish-wiki/wiki). If you would like to submit documentation to be included in the Wiki, please review the recommendations below and the steps to open an issue and submit a Pull Request below.
+
+## Style Guide
+When submitting a Markdown file for consideration for the Wiki, we ask you to follow the style guide outlined here, this will ensure consistency between all of our wiki pages.
+
+### Markdown Formatting
+Use [GitHub's Basic Formatting Syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings) as a guide on how to format text with Markdown, as GitHub does not support all general Markdown formatting you may find in other guides.
+
+### Assets
+When taking a screenshot or screen recording, all modules should be disabled. The wiki is intended for core System functionality only.
+When adding an image or video, first upload the asset to the original Issue created for the update. This will provide us with a reliable link to that asset for use between your Fork and the dnd5e system.  
+- Images
+    - I dunno, any guidelines here?
+    - Recommended software for taking and annotating screenshots?
+- Videos
+    - Webm preferred? Does is matter?
+    - Recommended software to capture video?
```

Comment:
Similar to above but worth mentioning that GH has a 10M file upload limit so it's worth mentioning that, and that keeping the dimensions of the video small, such as only the size of the dialog box you are demoing is a good way to keep within the 10M limit.

Reply:
Added information about the size limit, thanks for the info!